src/Raven.Studio/wwwroot/Content/css/pages/indexes.less

```diff
@@ -64,9 +64,12 @@
             }
         }
 
+        .clear-left {
```

Comment:
same here.

Reply:
1. moved to _styles-common.less_
2. used in _indexes.html_ to get the new icons in place

Reply:
ok